multi_import/data.py

```diff
@@ -228,7 +228,7 @@ def get_file(self, file_format: FileFormat=None, export_items_as_individual_file
 
         return file
 
-    def get_http_response(self, file_format: FileFormat=None, filename: str=None) -> HttpResponse:
+    def get_http_response(self, file_format: FileFormat=None, filename: str=None, export_items_as_individual_files: bool=False) -> HttpResponse:
```

Comment:
Sorry... I should've been clear lol... I meant even if we don't use the param in SDE, we should probably have the param working here? Like past it down to `get_file` or the other `get_http_response` call.